utils.js

```diff
@@ -0,0 +1,22 @@
+// class ProductArray {
+//     constructor(name, id, img) {
+//         this.name = name;
+//         this.id = id;
+//         this.img = img;
+//     }
+export function getProductsById(someId) {
+    let productMatch;
+    this.products.forEach(product => {
+        if (someId === product.id) {
+            productMatch = product;
+        }
+    });
+    return productMatch;
+}
+
+export function getRandomProduct() {
```

Comment:
This getRandomProduct function looks good. You should move it though into your ProductArray.js file and have it living as a method on your class